docs/source/dev_guide/algorithms/svm_multi_class/multi-class-classifier.rst

```diff
@@ -158,7 +158,7 @@ Prediction
 At the prediction stage, a multi-class classifier has the following parameters:
 
 .. list-table::
-   :widths: 10 20 30 25
+   :widths: 10 10 10 30
```

Comment:
Could you please fix following as well in this table? 
" -  ``voteBasedthe`` method based on the votes obtained from two-class classifiers."

Reply:
thanks, fixed